static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -1267,27 +1293,13 @@ var CStudioForms = CStudioForms || function() {
             serviceUrl += '&unlock=true';
           }
 
-          var createDialog = function () {
-            var dialogEl = document.getElementById('saveDraftWar');
-            if (!dialogEl) {
-              var dialog = new YAHOO.widget.SimpleDialog('saveDraftWar',
-                {
-                  width: '300px', fixedcenter: true, visible: false, draggable: false, close: true, modal: true,
-                  text: 'Draft Save Completed', icon: YAHOO.widget.SimpleDialog.ICON_INFO,
-                  constraintoviewport: true
-                });
-              dialog.setHeader('Notification');
-              dialog.render(document.body);
-              dialogEl = document.getElementById('saveDraftWar');
-              dialogEl.dialog = dialog;
-            }
-            dialogEl.dialog.show();
-            setTimeout(function () {
-              dialogEl.dialog.destroy();
-            }, 1500);
-          };
-
           if (me.config.isInclude) {
+            messages$.subscribe((message) => {
```

Comment:
Make sure subscription is created before the message to update. 
You may even subscribe to this on initialization time; specially since you're not unsubscribing.